api/src/main/java/org/openmrs/util/PrivilegeConstants.java

```diff
@@ -880,4 +880,14 @@
 	
 	@AddOnStartup(description = "Able to manage the search index")
 	public static final String MANAGE_SEARCH_INDEX = "Manage Search Index";
+	
+	@AddOnStartup(description = "Able to add an order set")
+	public static final String ADD_ORDERSET = "Add OrderSet";
+	
+	@AddOnStartup(description = "Able to edit the order set")
+	public static final String EDIT_ORDERSET = "Edit OrderSet";
+	
+	@AddOnStartup(description = "Able to get order sets")
+	public static final String GET_ORDERSET = "Get OrderSet";
```

Comment:
All these privileges should be named like `GET_ORDER_SET = "Get Order Set"` as is done in other examples in this file.